client/app/pages/queries/query.html

```diff
@@ -74,6 +74,22 @@
             </a>
         </span>
       </h3>
+      <div class="compare-query-version">
+        <span class="zmdi zmdi-archive"></span>
+        <span class="text-muted">Version</span>
+        <strong>{{query.version}}</strong>
+        &gt;
+        <span>Compare to</span>
+        <select id="version-choice">
+          <option ng-repeat="version in versions" value="{{version.object_version}}">
+            {{version.object_version}}
+          </option>
+        </select>
+        <button class="btn btn-s btn-default" ng-click="compareQueryVersion()">
+          <span class="zmdi zmdi-play"></span>
+          Compare
+        </button>
+      </div>
```

Comment:
Hrm. I do like the UI that you're proposing, @arikfr. There are some cases where it might be a bit unwieldy.. if a query is >100 lines of code and has multiple significant edits then the single page stream might be a bit long, but even so we can probably work around that by having the diffs be collapsible or something.

Might you be open to proceeding with our current UI and then treating this change as a separate issue?